### ISSUE TITLE
Build fails on m1

### ISSUE THREAD

**Issue by Ria8651** [jackmott/rust-simd-noise#40]
**Error**
```
error: the feature named `sse4.1` is not valid for this target
   --> /Users/brian/.cargo/registry/src/github.com-1ecc6299db9ec823/simdnoise-3.1.6/src/sse41.rs:386:18
    |
386 | #[target_feature(enable = "sse4.1")]
    |                  ^^^^^^^^^^^^^^^^^ `sse4.1` is not valid for this target
```
The console is spammed with this past the console overflow. There are similar errors for sse2 and avx2. There's probably not much you can do about it but I'll make an issue anyway

**Comment by merlinfuchs** [comment]
This has been fixed in the master branch and version 3.1.7 which isn't on crates.io yet.
Just use
```toml
simdnoise = { git = "https://github.com/jackmott/rust-simd-noise" }
```

**Comment by Ria8651** [comment]
Thanks :)
